docs/compatible_configs.md

```diff
@@ -0,0 +1,261 @@
+# _Black_ compatible configurations
+
+Most of the changes that _Black_ make are harmless, but a few do conflict against other
```

Comment:
All of Black's changes are harmless (or at least, they should be)

Reply:
Well noted, I think I've been stalking the issue section a bit too much.